src/constants.js

```diff
@@ -67,8 +67,10 @@ const UNPATCHED_MATH_METHODS = Object.getOwnPropertyNames(Math).filter((el) => {
 });
 
 const SHARED_SINGLE_OPS = {
+  "+": "add",
   "*": "mul",
   "-": "sub",
+  "/": "div",
```

Comment:
This causes problems with `div` throwing in Big Decimal mode — e.g., it means it does not. Livewise, in the transforms `add` is not necessary becase it is transformed into `addOrConcat`. Let's not change it here, but you can import the base levels and then `add` and `div` to the spread object like we do in `Decimal128`.

Reply:
Oops, I thought it was just a duplication that had gotten out of sync. (Well, I guess that's why we should get the tests working again :stuck_out_tongue:) I didn't mean to open another can of worms, sorry.